controllers/bsl.go

```diff
@@ -398,9 +399,9 @@ func (r *DPAReconciler) validateGCPBackupStorageLocation(bslSpec velerov1.Backup
 	return nil
 }
 
-func pluginExistsInVeleroCR(configuredPlugins []oadpv1alpha1.DefaultPlugin, expectedPlugin oadpv1alpha1.DefaultPlugin) bool {
+func pluginExistsInVeleroCR(configuredPlugins []oadpv1alpha1.DefaultPlugin, expectedProvider string) bool {
```

Comment:
This is a duplication of the function changed in `controllers/vsl.go`. I would delete function in that file and use this one

This function name is wrong, no? From what I see in code, DPA was called velero at some point in time. It should be `pluginExistsInDPA` or something like this, right?

Reply:
I was trying to avoid making major functionality changes since we want to backport to 1.4. The only change here is we're passing a regular string rather than a string that's typecast to `expectedPlugin` -- until this change, the plugin name and the provider were always the same. Now they're not, since both `legacy-aws` and `aws` have the provider `aws`.